app/api/public/akatsuki.py

```diff
@@ -0,0 +1,31 @@
+"""\
+Provides an API exposing Akatsuki's beatmaps, which
+include internal state such as ranked status updates.
+"""
+
+import logging
+
+from fastapi import APIRouter
+from fastapi import Header
+from fastapi import Response
+
+from app.api.responses import JSONResponse
+from app.usecases import akatsuki_beatmaps
+
+router = APIRouter(tags=["Akatsuki Beatmaps"])
+
+
+@router.get(
+    "/api/akatsuki/v1/beatmaps/akatsuki_beatmaps_lookup",
```

Comment:
hmm we should use a path that represents the function being invoked, atm this says nothing about "custom ranked beatmaps"

Reply:
also `-` is more common/standard in REST paths than `_`

perhaps
```suggestion
    "/api/akatsuki/v1/custom-ranked-beatmaps",
```

Reply:
_Ideally_ this API would actually be merged into a more generic `/api/akatsuki/v1/beatmaps` where we have a filter parameter like `/api/akatsuki/v1/beatmaps?only_custom_ranked=true` to _optionally_ filter on custom ranked status, otherwise returning all beatmaps (without the filtration) to make the API more flexible; I might make that change later